app/views/devise/sessions/new.html.erb

```diff
@@ -1,11 +1,5 @@
-</html> 
-<head>
-  <link href='https://fonts.googleapis.com/css?family=Graduate' rel='stylesheet' type='text/css'>
-  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-</head>
-
-<body>
   <div class="background"></div>
+  <%= yield %>
```

Comment:
<%= yield %> should only be in your layout templates (in this case 'application.html.erb'), so you can take this out :)